shipClass.py

```diff
@@ -0,0 +1,67 @@
+import math
+
+class Ship:
+
+    def __init__(self, pos = None, direct = None, thrust = None, speed = None):
+        """init for ships and other objects aswell"""
+        if(pos is None):
+            pos = [0,0]
+        if(direct is None):
+            direct = [1,0]
+        if(thrust is None):
+            thrust = 0
+        if(speed is None):
+            speed = [0,0]
+
+
+        self.pos = pos
+        self.direct = direct
+        self.thrust = thrust
+        self.speed = speed
+        self.cargo = ["cat"]                #   Stowaway cat
+
+    def __str__(self):
+        return "Hello. I am a spaceship."   #   Because I can that's why
+
+    def rotatecw(self, angle = 90):
+        """rotates the direction o the ship by the given degrees
+        note direction is in xy"""
+        angle = deg2rad(angle)
+        x = self.direct[0]
+        y = self.direct[1]
+        newx = math.cos(angle) * x + math.sin(angle) * y
+        newy = - math.sin(angle) * x + math.cos(angle) * y
+        self.direct = [newx, newy]
+
+    def add_cargo(self, item):              #   This is necessary
```

Comment:
Fun comments can be fun while you're writing code, but they can be confusing when someone else is reading it. Remember to sanitize.